src/components/register/verify.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { Redirect } from 'react-router-dom';
+import verifyEmail from '../../actions/verifyAction';
+
+const verify = (props) => {
```

Comment:
Learn to name your components using Capitalized case, Like `Verify` or `EmailVerify`